### DISCUSSION TITLE
Code Highlighting does not work..

### DISCUSSION THREAD

**Discussion by jaysekar** [mkdocs/mkdocs#3825] [Q&A] [answered]
I am not sure why my code highlighting is not working. I have this in the mkdocs.yml

  features: 
    - navigation.instant
    - navigation.instant.progress
    - navigation.instant.preview
    - navigation.tracking
    - navigation.tabs
    - navigation.tabs.sticky
    - navigation.sections
    - navigation.expand
    #- toc.integrate
    - navigation.top
    - search.suggest
    - search.highlight
    - search.share
    - content.code.annotation
    - content.code.copy
  
  markdown_extensions:
    - pymdownx.highlight:
        anchor_linenums: true
    - pymdownx.inlinehilite
    - pymdownx.snippets
    - admonition
    - pymdownx.arithmatex:
        generic: true
    - footnotes
    - pymdownx.details
    - pymdownx.superfences
    - pymdownx.mark
    - attr_list

In my python.md file, I have this.

Here's some Python Code

``` py 
def bubble_sort(items):
    for i in range(len(items)):
        for j in range(len(items) - 1 - i):
            if items[j] > items[j + 1]:
                items[j], items[j + 1] = items[j + 1], items[j]

```

This shows up as a code block but no highlighting.

**Comment by Andre601** [comment]
Could you share that mkdocs.yml snippet as code block, so that it actually displays properly?

**Reply by kamilkrzyskow** [reply, marked as answer]
Yes, send the actual `mkdocs.yml` file here or whole zip archive as [a reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/#creating-a-reproduction)

I'm assuming the OP is using `plugins:` and `markdown_extensions:` under the `theme:` just like `features:`, but that's incorrect.
`plugins:` and `markdown_extensions:` should be on the same level as the `theme:`, as those are MkDocs configuration settings:
https://www.mkdocs.org/user-guide/configuration/

**Comment by jaysekar** [comment]
Hi there, you are correct. It was an indentation issue. I corrected the indent for both the markdown extensions and the plugins and they both work correctly now. Thank You.

[mkdocs.yml.txt](https://github.com/user-attachments/files/16840539/mkdocs.yml.txt)!
